fractions/javaee/cdi/src/main/java/org/wildfly/swarm/cdi/runtime/CDIArchivePreparer.java

```diff
@@ -0,0 +1,41 @@
+package org.wildfly.swarm.cdi.runtime;
+
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.Node;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.wildfly.swarm.spi.api.ArchivePreparer;
+import org.wildfly.swarm.spi.api.JARArchive;
+
+/**
+ * @author Bob McWhirter
+ */
+public class CDIArchivePreparer implements ArchivePreparer {
+
+    public static final String META_INF_BEANS_XML = "META-INF/beans.xml";
+    public static final String WEB_INF_CLASSES_BEANS_XML = "WEB-INF/classes/beans.xml";
```

Comment:
Ah, I was going from http://stackoverflow.com/questions/8991561/where-should-beans-xml-be-placed